evap/static/ts/src/csrf-utils.ts

```diff
@@ -1,16 +1,5 @@
-// based on: https://docs.djangoproject.com/en/3.1/ref/csrf/#ajax
```

Comment:
Since this is not based on cookies anymore, can you revert all changes that were related to making `getCookie` available for your code? `csrf-utils.ts` and `utils.ts` should have only minimal changes, then.